From andrea.gelmini@gelma.net  Tue Jun 15 11:58:49 2010
From: Andrea Gelmini <andrea.gelmini@gelma.net>
Date: Sun, 23 May 2010 21:56:43 +0200
Subject: USB: Documentation/usb/linux.inf: Checkpatch cleanup
To: andrea.gelmini@gelma.net
Cc: Greg Kroah-Hartman <gregkh@suse.de>, Randy Dunlap <rdunlap@xenotime.net>, linux-usb@vger.kernel.org, linux-doc@vger.kernel.org
Message-ID: <1274644605-25181-47-git-send-email-andrea.gelmini@gelma.net>


Documentation/usb/linux.inf:66: ERROR: trailing whitespace
Documentation/usb/linux.inf:67: ERROR: trailing whitespace
Documentation/usb/linux.inf:98: ERROR: trailing whitespace
Documentation/usb/linux.inf:99: ERROR: trailing whitespace
Documentation/usb/linux.inf:115: ERROR: trailing whitespace
Documentation/usb/linux.inf:116: ERROR: trailing whitespace
Documentation/usb/linux.inf:120: ERROR: trailing whitespace
Documentation/usb/linux.inf:121: ERROR: trailing whitespace
Documentation/usb/linux.inf:122: ERROR: trailing whitespace
Documentation/usb/linux.inf:123: ERROR: trailing whitespace
Documentation/usb/linux.inf:144: ERROR: trailing whitespace
Documentation/usb/linux.inf:145: ERROR: trailing whitespace
Documentation/usb/linux.inf:146: ERROR: trailing whitespace
Documentation/usb/linux.inf:147: ERROR: trailing whitespace
Documentation/usb/linux.inf:158: ERROR: trailing whitespace

Signed-off-by: Andrea Gelmini <andrea.gelmini@gelma.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/usb/linux.inf |   30 +++++++++++++++---------------
 1 file changed, 15 insertions(+), 15 deletions(-)

--- a/Documentation/usb/linux.inf
+++ b/Documentation/usb/linux.inf
@@ -63,8 +63,8 @@ HKR, Ndi,            DeviceId,         0
 CopyFiles=RNDIS_CopyFiles_98
 
 [RNDIS_CopyFiles_98]
-usb8023.sys, usb8023w.sys, , 0 
-rndismp.sys, rndismpw.sys, , 0 
+usb8023.sys, usb8023w.sys, , 0
+rndismp.sys, rndismpw.sys, , 0
 
 ; Windows Millennium Edition specific sections --------------------
 
@@ -95,8 +95,8 @@ HKR, Ndi,            DeviceId,         0
 CopyFiles=RNDIS_CopyFiles_ME
 
 [RNDIS_CopyFiles_ME]
-usb8023.sys, usb8023m.sys, , 0 
-rndismp.sys, rndismpm.sys, , 0 
+usb8023.sys, usb8023m.sys, , 0
+rndismp.sys, rndismpm.sys, , 0
 
 ; Windows 2000 specific sections ---------------------------------
 
@@ -112,15 +112,15 @@ AddService = USB_RNDIS, 2, RNDIS_Service
 
 [RNDIS_CopyFiles_NT]
 ; no rename of files on Windows 2000, use the 'k' names as is
-usb8023k.sys, , , 0 
-rndismpk.sys, , , 0 
+usb8023k.sys, , , 0
+rndismpk.sys, , , 0
 
 [RNDIS_ServiceInst_NT]
 DisplayName     = %ServiceDisplayName%
-ServiceType     = 1 
-StartType       = 3 
-ErrorControl    = 1 
-ServiceBinary   = %12%\usb8023k.sys    
+ServiceType     = 1
+StartType       = 3
+ErrorControl    = 1
+ServiceBinary   = %12%\usb8023k.sys
 LoadOrderGroup  = NDIS
 AddReg          = RNDIS_WMI_AddReg_NT
 
@@ -141,10 +141,10 @@ AddService      = USB_RNDIS, 2, RNDIS_Se
 
 [RNDIS_ServiceInst_51]
 DisplayName     = %ServiceDisplayName%
-ServiceType     = 1 
-StartType       = 3 
-ErrorControl    = 1 
-ServiceBinary   = %12%\usb8023.sys    
+ServiceType     = 1
+StartType       = 3
+ErrorControl    = 1
+ServiceBinary   = %12%\usb8023.sys
 LoadOrderGroup  = NDIS
 AddReg          = RNDIS_WMI_AddReg_51
 
@@ -155,7 +155,7 @@ HKR, , MofImagePath, 0x00020000, "System
 
 [RNDIS_AddReg_NT]
 HKR, Ndi,               Service,        0, "USB_RNDIS"
-HKR, Ndi\Interfaces,    UpperRange,     0, "ndis5" 
+HKR, Ndi\Interfaces,    UpperRange,     0, "ndis5"
 HKR, Ndi\Interfaces,    LowerRange,     0, "ethernet"
 
 [RNDIS_EventLog]
